docs/Errors.rst

```diff
@@ -64,6 +64,11 @@ Server value errors (``10000``–``10099``)
 - ``10050``: “The server value with name \____\_ is not an LLVM method
    specification”
    ``{ name: String }``
+- ``10080``: “The server value with name \____\_ is not an JVM class”
```

Comment:
I think this should be "a JVM class" rather than "an JVM class"

Reply:
Fixed.